### PR TITLE
Fixed asian chart

### PR DESCRIPTION
**Pull request by 4p00rv**

### CODE REVIEW

File: src/trade/tradeConf.js [hunk 1]
```diff
@@ -240,7 +240,7 @@ define(['lodash', 'jquery', 'moment', 'websockets/binary_websockets', 'common/ri
               if(this.category === 'Asians') {
                 //https://trello.com/c/ticslmb4/518-incorrect-decimal-points-for-asian-average
                 var avg = this.average().toFixed(decimal_digits + 1);
-                return {value: avg, label:'Average ('.i18n() + avg + ')', id: 'plot-barrier-y'};
+                return {value: avg*1, label:'Average ('.i18n() + avg + ')', id: 'plot-barrier-y'};
```
Comment (4p00rv) [inline src/trade/tradeConf.js]: @arnabk Although I have fixed this but I don't understand why do we need to multiply the value with 1. Can you help?

**Comment by 4p00rv** [comment]
@arnabk #523  also fixes the same issue. Please merge any one of these.

File: src/trade/tradeConf.js (hunk 1 above)
Comment (binarytom) [inline src/trade/tradeConf.js]: `.toFixed()` returns a string, the *1 converts it back into a number. If you want to make the behaviour clearer, you could use `parseFloat(avg)` instead.

Comment (4p00rv) [inline src/trade/tradeConf.js]: @binarytom thanks for the explanation. It makes sense now :+1:

**Pull request closed without merging**